lib/Sema/CSSimplify.cpp

```diff
@@ -2863,6 +2863,46 @@ getArgumentLabels(ConstraintSystem &cs, ConstraintLocatorBuilder locator) {
   return known->second;
 }
 
+
+/// Return true if the specified type or a super-class/super-protocol has the
+/// @dynamicMemberLookup attribute on it.  This implementation is not
+/// particularly fast in the face of deep class hierarchies or lots of protocol
+/// conformances, but this is fine because it doesn't get invoked in the normal
+/// name lookup path (only when lookup is about to fail).
+static bool isDynamicMemberLookupable(Type ty) {
```

Comment:
Can I suggest `conformsToDynamicMemberLookup`?

Reply:
@rudkx But can you *conform* to an attribute?

I suggest `hasDynamicMemberLookup`.

Reply:
Thanks, renamed to hasDynamicMemberLookupAttribute